src/ingest/orchestrator.py

```
@@ -0,0 +1,447 @@
+            job_type=job_type or "json",
+            job_data=job_data,
+            priority=0,
+            retry_count=0,
+            max_retries=3,
+            created_at=datetime.utcnow()
+        )
+        
+        try:
+            self.queue.enqueue(queue_message)
+            self.db.commit()
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)